### DISCUSSION TITLE
BLE Beaconing in node-matter?

### DISCUSSION THREAD

**Discussion by javierbarellano4v** [mfucci/node-matter#236] [Q&A] [answered]
Does node-matter support/implement BLE beaconing?

**Comment by Apollon77** [comment]
Right now we initially focus on "normal Ethernet", BLE and Thread are planned later

**Reply by javierbarellano4v** [reply]
@Apollon77 Ok. Gotcha.

**Comment by mfucci** [comment, marked as answer]
See https://github.com/mfucci/node-matter/discussions/37
Adding BLE support will introduce platform dependencies, so should be done into a separate repository. As a short term solution, I was suggesting to do it in a separate branch
